pkg/kubelet/secret/secret_manager_test.go

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
This file is effectively a copy of part of previously existing test (but run explicitly on cachedBasedSecretmanager instead of cacheBasedManager).